src/env/uat/terraform.tfvars

```diff
@@ -67,11 +67,12 @@ postgresql_network_rules = {
   # dblink
   allow_access_to_azure_services = false
 }
-prostgresql_db_mockpsp = "mock-psp"
+prostgresql_db_mockpsp    = "mock-psp"
+postgresql_schema_mockpsp = "mock_psp_uat"
```

Comment:
Can we call schema mock_psp without env? in this way we reduce differences between envs

Reply:
right 👍

Reply:
✅

Reply:
I updated postgress module, and explicitly added `privatelink_postgres_database_azure_com` and `privatelink_postgres_database_azure_com_vnet` (I think there were some changes made on the portal), @pasqualedevita  feedback welcome:-)